src/views/catalog/customize/components/FormActionGroup.tsx

```diff
@@ -16,16 +16,17 @@ type FormActionGroupProps = {
   }>;
 };
 
-export const FormActionGroup: React.FC<FormActionGroupProps> = ({ loading, onCancel }) => {
+export const FormActionGroup: FC<FormActionGroupProps> = ({ loading, onCancel }) => {
   const { t } = useKubevirtTranslation();
   const { ns: namespace } = useParams<{ ns: string }>();
   const history = useHistory();
+  const namespaceUrl = namespace ? `ns/${namespace}` : 'all-namespaces';
```

Comment:
Hi :) 
Please use the const ALL_NAMESPACES instead of hard-coded 'all-namespaces'

Reply:
Also since namespaceUrl is used only inside the handleCancel function, you can either define it inside of handleCancel or inline this as ternary in the push, to avoid adding it to dependencies array

Reply:
You are totally right, thanks!